publisher/product/ontology/build-about.sh

```diff
@@ -86,10 +78,10 @@ __HERE__
     "${JENA_ARQ}" \
       --data="${tmpAboutFileDev}" \
       --query="${SCRIPT_DIR}/lib/echo.sparql" \
-      --results=RDF > "${tag_root}/LoadFIBODev.rdf" 2> "${TMPDIR}/err.tmp"
+      --results=RDF > "${tag_root}/Load${ONTPUB_FAMILY}Dev.rdf" 2> "${TMPDIR}/err.tmp"
```

Comment:
Should be uppercase:
`--results=RDF > "${tag_root}/Load${ONTPUB_FAMILY^^}Dev.rdf" 2> "${TMPDIR}/err.tmp"`